example/stories/examples/mocks.stories.js

```diff
@@ -0,0 +1,58 @@
+import { withAngularJs } from "storybook-addon-angularjs";
+
+import mockTestApp from "../../src/mock-test.module";
+
+export default {
+  title: "Mocks",
+};
+
+/**
+ * Story with template and service mock
+ */
+export const MockComponent = () => ({
+  template: /* HTML */ ` <mock-test></mock-test> `,
+});
+
+class MockedService {
+  constructor() {
+    console.log("[MockedService] New Instance");
+    this.counter = 0;
+  }
+
+  increaseCounter() {
+    this.counter++;
+  }
+}
+
+function MockedServiceFromFactory() {
+  let counter = 0;
+
+  return {
+    counter,
+    increaseCounter: () => {
+      return ++counter;
+    },
```

Comment:
What do you think of this?

```suggestion
    get counter(){
      return counter;
    },
    increaseCounter: () => {
      return counter++;
    },
```

This allows the `MockedServiceFromFactory` to be used just like `MockedService` 

```html
<button ng-click="$ctrl.mockedServiceFromFactory.increaseCounter()">Increase Count</button>
<div>{{$ctrl.mockedServiceFromFactory .counter}}</div>
```